gh-actions/appauth/.eslintrc.json

```diff
@@ -4,7 +4,6 @@
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
-    "prettier/@typescript-eslint",
```

Comment:
can you explain more why to remove this?

i think if we are going to make these kind of changes probs best do those next separately - but would like to understand better why to remove

Reply:
It has been deprecated and included in "prettier".